tests/ft_rust.vader

```diff
@@ -0,0 +1,153 @@
+Include: include/setup.vader
+
+Execute (cargo skip non json line):
+  runtime autoload/neomake/highlights.vim
+
+  file src/lib.rs
```

Comment:
better to use `new` before, and `bwipe` after - to not mess with buffers having a name etc.